src/css/style.css

```diff
@@ -1,3 +1,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.checkbox:checked {
+  background: linear-gradient(to right, hsl(192deg 100% 67%), hsl(280deg 87% 65%));
+}
```

Comment:
we should remove this here and use the tailwind config

Reply:
Done